src/game/client/components/hud.cpp

```diff
@@ -1259,12 +1260,12 @@ inline float CHud::GetMovementInformationBoxHeight()
 	return BoxHeight;
 }
 
-void CHud::UpdateMovementInformationTextContainer(STextContainerIndex &TextContainer, float FontSize, float Value, char *pPrevValue, size_t Size)
+void CHud::UpdateMovementInformationTextContainer(STextContainerIndex &TextContainer, float FontSize, float Value, char *pPrevValue, size_t Size, bool ForceUpdate)
```

Comment:
```suggestion
void CHud::UpdateMovementInformationTextContainer(STextContainerIndex &TextContainer, float FontSize, float Value, char *pPrevValue, size_t Size)
```
It's unclear what `ForceUpdate` is supposed to be, and it's probably an antipattern. The function should recognize that something changed and update it based on that.